grammar/Makefile

```diff
@@ -0,0 +1,9 @@
+ANTLR_JAR=antlr-4.13.2-complete.jar
```

Comment:
let's put this in a go generate block instead of adding make to the project.

Reply:
For example, a block something like this should work:

```
//go:generate wget -nc https://www.antlr.org/download/antlr-4.11.1-complete.jar
//go:generate -command antlr java -Xmx500M -cp "./antlr-4.11.1-complete.jar:$CLASSPATH" org.antlr.v4.Tool
//go:generate antlr -Dlanguage=Go -package substrait bar/foo.g4 -Werror -long-messages -o ./
```

Reply:
Fixed it.